service/messageHandler_test.go

```diff
@@ -315,17 +216,17 @@ func TestMessageProducerError(t *testing.T) {
 
 	// check of monitoring logging
 	logTest.Assert(t, hook.LastEntry()).
-		HasMonitoringEvent("Map", testTxId, "Annotations").
+		HasMonitoringEvent("Map", testTxID, "Annotations").
 		HasValidFlag(true).
-		HasUUID(contentUuid)
+		HasUUID(contentUUID)
 }
 
 func TestNilWhitelistIsIgnoredWithErrorLog(t *testing.T) {
 	hook := logTest.NewTestHook("test")
 	mp := &mockMessageProducer{}
 	service := NewAnnotationMapperService(nil, mp)
 	inbound := kafka.FTMessage{
-		Headers: map[string]string{"Origin-System-Id": testSystemId},
+		Headers: map[string]string{"Origin-System-Id": testSystemID},
 		Body:    `{"foo":"bar"}`,
 	}
 
```

Comment:
As far as I know, checking error strings in tests is not a good practice. Can we remove the assertion on line 238?

Reply:
You are right. But if I remove that assertion the test becomes pointless. And I would need to change `HandleMessage` interface to rewrite this test. It can be done, but I try to touch as little code as possible.